plugins/inputs/influxdb_listener/influxdb_listener.go

```diff
@@ -127,7 +126,6 @@ func (h *InfluxDBListener) Init() error {
 	tags := map[string]string{
 		"address": h.ServiceAddress,
 	}
-	h.bytesRecv = selfstat.Register("influxdb_listener", "bytes_received", tags)
```

Comment:
Why do you remove this stat? It is definitively useful. :-)

Reply:
I agree that it is useful. However, there is no longer a way to get the current position of the parser, so we can't track that stat as we used to. We could read the entire body into memory and get its length, but that is far from an ideal solution.

Reply:
So how about using the `req.ContentLength` or similar to approximate that number?

Reply:
@akrantz01 do you think you have the time to resolve Sven's suggestion here? No worries if you don't, we can take over the pr if you don't time.

Reply:
Sorry about the delay. Using `req.ContentLength` should work as a decent approximation.